### DISCUSSION TITLE
Additional 3-Phase Distinctions (A-B, B-C, C-A)

### DISCUSSION THREAD

**Discussion by iunderwood** [netbox-community/netbox#8028] [Ideas]
I'm working on a new MDF / Data Facility and as a design consideration, we implemented 3-phase power as 208 Delta.

The current documentation capability describes feed legs A, B, and C.  This would be appropriate for a 3-Phase Wye where current would be drawn from only one of the phases.  To support my use case, I'd like to see "A-B", "B-C", "C-A" added as phase leg options.

i did want to get some feedback before submitting this as a feature request / issue.

**Comment by bluikko** [comment]
I have wondered about the same, having one facility with 3-phase power. In the end I just resigned to using the standard A, B, C since it could be thought of as a notion for a wye/delta setup.

But someone else might have both delta and wye setups and then there is no way to distinguish them without custom fields.

**Reply by iunderwood** [reply]
It does bear some consideration because the load on 3-phase is distributed between the phases themselves.  Just to keep the math simple, a 208w device on a 208v circuit on B-C will have a measured draw on 1A measured on both B and C.

It's a narrow use case since I don't expect the user base gets that deep into the weeds with power.

**Comment by mmfreitas** [comment]
We use L1, L2 and L3 as identification of the feed leg. I think to support every each case would be either to make the phase field choices customizable, or to use a custom field for that...
